src/views/RequestTypeForm/index.tsx

```diff
@@ -31,31 +33,42 @@ const RequestTypeForm = () => {
     }
   );
 
-  const handleCreateIntake = (values: { requestType: string }) => {
+  const handleIntakeCreationRedirect = (intake: SystemIntakeForm) => {
```

Comment:
This is our function to invoke on success. It's the routing. This routing only occurs with a successful API request.